app/cafe.py

```diff
@@ -0,0 +1,24 @@
+import datetime
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
```

Comment:
You should put a blank line between each group of imports.

Reply:
done